frontend/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# python:3.6-alpine3.9 specific working version
+FROM python@sha256:ad2b24eacc9865244d7b1ae198d507f4ce8f506484c35d27158e7f1dd6041a8b
+
+ENV ENV="development"
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ENV WORKON_HOME="/.venv"
+
+COPY ./docker-entrypoint.sh /
```

Comment:
This can be removed by changing `ENTRYPOINT` to `["/src/sample-app/frontend/docker-entrypoint.sh"]`